modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/shared/select-tasks-step/SelectTasksStepTable.es.js

```diff
@@ -38,7 +39,14 @@ function Item({totalCount, ...task}) {
 	return (
 		<ClayTable.Row className={checked ? 'table-active' : ''}>
 			<ClayTable.Cell>
-				<ClayCheckbox checked={checked} onChange={handleCheck} />
+				<ClayCheckbox
+					aria-label={sub(
+						Liferay.Language.get('select-x'),
+						`${assetType}: ${assetTitle}`
```

Comment:
We should probably make this fully translatable by doing `select-x-x` and providing to variable to sub. Otherwise this may not get rendered correctly in RTL languages.

Reply:
Fixed